lib/manager/circleci/update.ts

```diff
@@ -1,11 +1,11 @@
-const { logger } = require('../../logger');
-const { getNewFrom } = require('../dockerfile/update');
+import { logger } from '../../logger';
+import { getNewFrom } from '../dockerfile/update';
+import { Upgrade } from '../common';
 
-module.exports = {
-  updateDependency,
-};
-
-function updateDependency(fileContent, upgrade) {
+export function updateDependency(
+  fileContent: string,
+  upgrade: Upgrade<{ lineNumber: number }>
```

Comment:
Are these additions necessary now that we moved lineNumber into managerData?

Reply:
no, the generic part is optional, just to help type check. i can remove that.

Reply:
should i remove that?

Reply:
I think for this one it's not necessary because lineNumber got moved to managerData